content/id/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,234 @@
+
+Dalam kasus sederhana, satu DaemonSet, mencakup semua node, akan digunakan untuk
+setiap jenis daemon. Pengaturan yang lebih rumit bisa saja menggunakan lebih
+dari satu DaemonSet untuk satu jenis daemon, tapi dengan _flag_ dan/atau
+`requests` cpu/memori yang berbeda untuk jenis `hardware` yang berbeda.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Menulis Spec DaemonSet
+
+### Buat DaemonSet
+
+Anda bisa definisikan DaemonSet dalam berkas YAML. Contohnya, berkas
+`daemonset.yaml` di bawah mendefinisikan DaemonSet yang menjalankan _image_ Docker
+fluentd-elasticsearch:
+
+{{< codenew file="controllers/daemonset.yaml" >}}
+
+* Buat DaemonSet berdasarkan berkas YAML:
+```
+kubectl apply -f https://k8s.io/examples/controllers/daemonset.yaml
+```
+
+### _Field_ Wajib
+
+Seperti semua konfigurasi Kubernetes lainnya, DaemonSet membutuhkan _field_
+`apiVersion`, `kind`, dan `metadata`.  Untuk informasi umum tentang berkas konfigurasi, lihat dokumen [deploying applications](/docs/user-guide/deploying-applications/),
+[pengaturan kontainer](/docs/tasks/), dan [pengelolaan objek dengan kubectl](/docs/concepts/overview/working-with-objects/object-management/).
+
+DaemonSet juga membutuhkan bagian [`.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+### Templat Pod
+
+`.spec.template` adalah salah satu _field_ wajib di dalam `.spec`.
+
+`.spec.template` adalah sebuah [templat pod](/docs/concepts/workloads/pods/pod-overview/#templat-pod). Skemanya benar-benar sama dengan [Pod](/docs/concepts/workloads/pods/pod/), kecuali bagian bahwa ia bersarang/_nested_ dan tidak memiliki `apiVersion` atau `kind`.
+
+Selain _field_ wajib untuk Pod, templat pod di DaemonSet harus
+menspesifikasikan label yang sesuai (lihat [selektor pod](#selektor-pod)).
+
+Templat pod di DaemonSet harus memiliki [`RestartPolicy`](/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy)
+yang bernilai `Always`, atau tidak dispesifikasikan, sehingga _default_ menjadi `Always`.
+
+### Selektor Pod
+
+_Field_ `.spec.selector` adalah selektor pod. Cara kerjanya sama dengan `.spec.selector` pada [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/).
+
+Pada Kubernetes 1.8, kamu harus menspesifikasikan pod selector yang cocok dengan label pada `.spec.template`.
+Selektor pod tidak akan lagi diberi nilai _default_ ketika dibiarkan kosong. Nilai _default_ selektor tidak
```

Comment:
all pod should be in capital, ensure same formatting for the rest of the word on this page
```suggestion
Selektor Pod tidak akan lagi diberi nilai _default_ ketika dibiarkan kosong. Nilai _default_ selektor tidak
```